packages/grafana-ui/src/components/Combobox/getComboboxStyles.ts

```diff
@@ -27,6 +27,7 @@ export const getComboboxStyles = (theme: GrafanaTheme2) => {
       position: 'relative',
       borderRadius: theme.shape.radius.default,
       overflow: 'hidden',
+      padding: theme.spacing(0.5),
```

Comment:
think we should put this `padding` on the `ScrollContainer` in combobox instead, else:
- the scroll indicator (shadow) doesn't properly span the full width of the menu
- you can't fully scroll the bottom element into view